### PR TITLE
Add camel-quartz wrapper

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
Add camel-quartz wrapper

### CODE REVIEW

**Comment by essobedo** [comment]
@ozgurkisir-qlik warning there is still a conflict to resolve

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -1588,13 +1588,14 @@
...
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.quartz-scheduler/quartz/${quartz-version}$overwrite=merge&amp;DynamicImport-Package=org.apache.camel.component.quartz</bundle>-->
-<!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
-<!--        <bundle>mvn:org.apache.camel/camel-quartz/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-quartz' version='${project.version}' start-level='50'>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>
+        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>
+        <feature version='${project.version}'>camel-core</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-core</feature>
```

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -1588,13 +1588,14 @@
...
-<!--        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.quartz-scheduler/quartz/${quartz-version}$overwrite=merge&amp;DynamicImport-Package=org.apache.camel.component.quartz</bundle>-->
-<!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
-<!--        <bundle>mvn:org.apache.camel/camel-quartz/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-quartz' version='${project.version}' start-level='50'>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>
+        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${project.version}'>camel-cron</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-cron</feature>
```

File: pom.xml [hunk 3]
```diff
@@ -564,6 +564,9 @@
         <geronimo-atinject.version>1.2</geronimo-atinject.version>
         <mongo-java-driver-version>4.11.1</mongo-java-driver-version>
         <velocity-version>2.3</velocity-version>
+        <quartz-version>2.3.2</quartz-version>
+        <c3p0-bundle-version>0.9.5.5_1</c3p0-bundle-version>
+        <hikaricp-version>2.4.13</hikaricp-version>
```
Comment (essobedo) [inline pom.xml]:
```suggestion
```
Let's use already existing versions

File: features/src/main/feature/camel-features.xml [hunk 4]
```diff
@@ -1588,13 +1588,14 @@
...
     </feature>
-<!--    <feature name='camel-quartz' version='${project.version}' start-level='50'>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.quartz-scheduler/quartz/${quartz-version}$overwrite=merge&amp;DynamicImport-Package=org.apache.camel.component.quartz</bundle>-->
-<!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
-<!--        <bundle>mvn:org.apache.camel/camel-quartz/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-quartz' version='${project.version}' start-level='50'>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Reuse the `${c3p0-version}` to build the version of the bundle by post-fixing it with `_1` corresponding to the version of the bundle

File: features/src/main/feature/camel-features.xml [hunk 5]
```diff
@@ -1588,13 +1588,14 @@
...
+        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: As we don't have a maven property `hikaricp-version`, let's put directly the version which is `2.4.13` according to the dependency:tree

Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

File: features/src/main/feature/camel-features.xml (hunk 4 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

File: pom.xml (hunk 3 above)
Comment (ozgurkisir-qlik) [reply pom.xml]: Done

File: features/src/main/feature/camel-features.xml [hunk 6]
```diff
@@ -1588,13 +1588,14 @@
...
-<!--        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/${hikaricp-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.quartz-scheduler/quartz/${quartz-version}$overwrite=merge&amp;DynamicImport-Package=org.apache.camel.component.quartz</bundle>-->
-<!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
-<!--        <bundle>mvn:org.apache.camel/camel-quartz/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-quartz' version='${project.version}' start-level='50'>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-version}_1</bundle>
+        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/2.4.13</bundle>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${camel.osgi.version.range}'>camel-cron</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-core</feature>
        <feature version='${camel.osgi.version.range}'>camel-cron</feature>
        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-version}_1</bundle>
        <bundle dependency='true'>mvn:com.zaxxer/HikariCP-java7/2.4.13</bundle>
```
Features first

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**